src/main/java/yalter/mousetweaks/forge/ConfigGui.java

```diff
@@ -35,6 +36,11 @@
 	                                                      Property.Type.STRING,
 	                                                      new String[]{ "Smooth scrolling, minor issues",
 	                                                                    "Non-smooth scrolling, no issues" });
+	private static Property scrollItemScaling = new Property("Scroll item scaling",
+															 "Relative to scroll amount",
+															 Property.Type.STRING,
+															 new String[]{ "Relative to scroll amount",
+																		   "Always exactly one item" });
```

Comment:
Hmm, I think this should be made clearer, but I'm not quite sure how to explain it properly. It's not really relative to scroll amount vs. always exactly one item because on most setups there's essentially no difference between the two.

At the very least I'd put a more thorough explanation into the tooltip, something like:

On some setups (notably macOS) scrolling the wheel with different speed results in different number of reported scroll ticks per one wheel "bump". To make them play nicely with Mouse Tweaks, set this option to "Always exactly one item".

Reply:
good point.